mysql-replication/azuredeploy.json

```diff
@@ -207,7 +212,8 @@
         "sshIPConfig": "[concat(variables('lbID'),'/frontendIPConfigurations/',variables('sshIPConfigName'))]",
         "nicName": "[concat(parameters('dnsName'), '-nic')]",
         "availabilitySetName": "[concat(parameters('dnsName'), '-set')]",
-        "vhdContainer": "[concat('http://',parameters('storageAccountName'),'.blob.core.windows.net/vhds/')]",
+        "storageAccountName": "[concat(parameters('storageAccountPrefix'), '0')]",
```

Comment:
@liupeirong Looks like this'll likely cause storage name collisions in case a user uses the default value. I suggest appending a random string using the uniqueString function